app/assets/stylesheets/_flash.css

```diff
@@ -0,0 +1,6 @@
+.flashes{
+   background-color: azure;
+   padding: 5px;
+   border: 1px solid blue;
+   width: fit-content;
+ }
```

Comment:
This file is probably leftover from the rebase and should be removed, because you have https://github.com/hooroo/domus/blob/master/app/assets/stylesheets/components/_flashes.scss instead

Reply:
Done in ```c01d0c0```.